.env.test

```diff
@@ -1,6 +1,8 @@
 DEBUG=true
 SECRET_KEY='tests'
 DATABASE_URL='postgres://localhost/opencraft'
+INSTANCE_MYSQL_URL='mysql://root@localhost'
+INSTANCE_MONGO_URL='mongodb://localhost'
```

Comment:
Reading your comment again I realise you might be talking about CI. If I remember correctly CircleCI provides mysql and mongodb on its standard containers, so it should be enough just to add the `INSTANCE_MYSQL_URL` and `INSTANCE_MONGO_URL` environment variables to the test environment.